### PR TITLE
add superlove & CFAA (otwarchive sites) support

### PR DESCRIPTION
**Pull request by xxwhirlpool**
this is my first pull request, please forgive any problems or inconveniences.

i added support for my site, [superlove](https://superlove.sayitditto.net/). it is otw-archive based, just as squidgworld and adastrafanfic, so i simply drew from squidgeworld's adapter and configuration to add support for my site. i tested downloads through the CLI and it worked.

please let me know if any changes should be made for further/necessary missing functionality. thank you.

### CODE REVIEW

**Comment by JimmXinu** [comment]
Looks good, but I don't have good internet for a few more days.  

Feel free to remind me if I haven't gotten back to this by, say, Tuesday next week.

**Comment by xxwhirlpool** [comment]
hey, with direct permission from the site admin, i also added support for another OTW-archive site, [CFAA](https://www.cfaarchive.org/). i tested the CLI and plugin and both work. changing PR title to reflect this.

File: fanficfare/fanficfare-custom-superlove.zip
Comment (JimmXinu) [inline fanficfare/fanficfare-custom-superlove.zip:1]: ??? Is this included by mistake?  Can you remove it?

File: fanficfare/adapters/adapter_cfaa.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#  -*- coding: utf-8 -*-
+
+# Copyright 2020 FanFicFare team
```
Comment (JimmXinu) [inline fanficfare/adapters/adapter_cfaa.py:3]: Copyright year should be set to current year for new files.

**Review by JimmXinu — CHANGES_REQUESTED** [review]
In future, please open a new PR for different sites.

File: fanficfare/adapters/adapter_cfaa.py (hunk 1 above)
Comment (xxwhirlpool) [reply fanficfare/adapters/adapter_cfaa.py:3]: noted, will update

File: fanficfare/fanficfare-custom-superlove.zip
Comment (xxwhirlpool) [reply fanficfare/fanficfare-custom-superlove.zip:1]: that was for testing the plugin on my end, 100% a mistake. sorry about that, will remove now.

**Comment by xxwhirlpool** [comment]
sorry about the mistakes. also noted about separating additional sites into their own PRs. i hope the fixes i've made are adequate and if not please tell me what else i should fix. again, very sorry

**Review by JimmXinu — APPROVED** [review]

**Pull request merged**